flow/include/flow/IConnection.h

```diff
@@ -179,13 +180,20 @@ class INetworkConnections {
 	// If a DNS name can be resolved to both and IPv4 and IPv6 addresses, we want IPv6 addresses when running the
 	// clusters on IPv6.
 	// This function takes a vector of addresses and return a random one, preferring IPv6 over IPv4.
+	// To prefer IPv4 addresses instead, set knob RESOLVE_PREFER_IPV4_ADDR to true.
 	static NetworkAddress pickOneAddress(const std::vector<NetworkAddress>& addresses) {
```

Comment:
Can we move this function to a .cpp file? IConnection.h is included by almost every file in FDB and a change would cause slowness during build.

Reply:
We can do it in a follow-up PR.